pkg/morph/deploy/funds.go

```diff
@@ -0,0 +1,232 @@
+package deploy
+
+import (
+	"context"
+	"fmt"
+	"math/big"
+
+	"github.com/nspcc-dev/neo-go/pkg/core/native"
+	"github.com/nspcc-dev/neo-go/pkg/core/transaction"
+	"github.com/nspcc-dev/neo-go/pkg/crypto/keys"
+	"github.com/nspcc-dev/neo-go/pkg/rpcclient/actor"
+	"github.com/nspcc-dev/neo-go/pkg/rpcclient/gas"
+	"github.com/nspcc-dev/neo-go/pkg/rpcclient/neo"
+	"github.com/nspcc-dev/neo-go/pkg/rpcclient/nep17"
+	"github.com/nspcc-dev/neo-go/pkg/rpcclient/notary"
+	"github.com/nspcc-dev/neo-go/pkg/smartcontract"
+	"github.com/nspcc-dev/neo-go/pkg/wallet"
+	"go.uber.org/zap"
+)
+
+const (
+	initialAlphabetGASAmount = 300
+	// lower threshold of GAS remaining on validator multi-sig account. It is needed
+	// to pay fees for transfer transaction(s). The value is big enough for
+	// transfer, and not very big to leave no tail on the account.
+	validatorLowerGASThreshold = 10
+)
+
+// groups parameters of makeInitialGASTransferToCommittee.
+type makeInitialGASTransferToCommitteePrm struct {
+	logger *zap.Logger
+
+	blockchain Blockchain
+
+	// based on blockchain
+	monitor *blockchainMonitor
+
+	committee keys.PublicKeys
+
+	localAcc             *wallet.Account
+	validatorMultiSigAcc *wallet.Account
+
+	tryTransfer bool
+}
+
+// makes initial transfer of funds to the committee for deployment procedure. In
+// the initial state of the Blockchain, all funds are on the validator multisig
+// account. Transfers:
+//   - 300GAS to each account of the Alphabet members
+//   - all other GAS to the committee multisig account
+//   - all NEO to the committee multisig account
+func makeInitialTransferToCommittee(ctx context.Context, prm makeInitialGASTransferToCommitteePrm) error {
```

Comment:
what exact problem does that commit solve? (we have access to everything we need, why transfer b/w accs?)